src/.parser.rs.pending-snap

```diff
@@ -0,0 +1,43 @@
+{"run_id":"1644797319-508675000","line":389,"new":{"module_name":"prql__parser__test","snapshot_name":"parse_table","metadata":{"source":"src/parser.rs","assertion_line":389,"expression":"parse(parse_to_pest_tree(r#\"table newest_employees = ( from employees )\"#,\n                         Rule::table).unwrap()).unwrap()"},"snapshot":"---\n- Table:\n    name: newest_employees\n    body:\n      - Pipeline:\n          - name: From\n            args:\n              - Ident: employees\n            named_args: []\n"},"old":{"module_name":"prql__parser__test","metadata":{},"snapshot":"---\n- Table:\n    name: newest_employees\n    body:\n      - name: From\n        args:\n          - Ident: employees\n        named_args: []"}}
```

Comment:
I think this is an unapproved snapshot file

Reply:
Thank you!